commit bbb4bee779b3301a3ab7924f85f38b96bdf1fd4a
Author: Takashi Iwai <tiwai@suse.de>
Date:   Thu Mar 10 11:33:43 2016 +0100

    ALSA: hda - Apply reboot D3 fix for CX20724 codec, too
    
    commit 56dc66ff1c6d71f9a38c4a7c000b72b921fe4c89 upstream.
    
    Just like CX20722, CX7024 codec also requires the power down at reboot
    in order to reduce the noise at reboot/shutdown.
    
    Bugzilla: https://bugzilla.kernel.org/show_bug.cgi?id=113511
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/pci/hda/patch_conexant.c b/sound/pci/hda/patch_conexant.c
index ef19890..600af58 100644
--- a/sound/pci/hda/patch_conexant.c
+++ b/sound/pci/hda/patch_conexant.c
@@ -204,8 +204,13 @@ static void cx_auto_reboot_notify(struct hda_codec *codec)
 {
 	struct conexant_spec *spec = codec->spec;
 
-	if (codec->core.vendor_id != 0x14f150f2)
+	switch (codec->core.vendor_id) {
+	case 0x14f150f2: /* CX20722 */
+	case 0x14f150f4: /* CX20724 */
+		break;
+	default:
 		return;
+	}
 
 	/* Turn the CX20722 codec into D3 to avoid spurious noises
 	   from the internal speaker during (and after) reboot */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1350_bbb4bee779b3301a3ab7924f85f38b96bdf1fd4a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1350_bbb4bee779b3301a3ab7924f85f38b96bdf1fd4a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
